tests/test_startup.py

```diff
@@ -71,3 +84,41 @@ async def test_async_startup_fail_on_bad_async_sqlalchemy_url(monkeypatch):
 
     with raises(Exception):
         await asyncio_support.startup()
+
+
+@mark.require_boto3
+@mark.dont_patch_engines
+def test_sync_startup_with_aws_rds_iam_enabled(
+    monkeypatch, boto_session, boto_client, db_host, db_user
+):
+    from fastapi_sqla import startup
+
+    monkeypatch.setenv("fastapi_sqla_aws_rds_iam_enabled", True)
+
+    with patch("boto3.Session", return_value=boto_session):
```

Comment:
Nit but why not patch in the `boto3_session` fixture?

Reply:
done